core/src/test/java/kafka/admin/ConfigCommandIntegrationTest.java

```diff
@@ -528,6 +529,17 @@ public void testUpdatePerBrokerConfigInKRaftThenShouldFail() {
         }
     }
 
+    @ClusterTest(types = {Type.CO_KRAFT, Type.KRAFT})
+    public void testIntervalMsParser() {
+        alterOpts = asList("--bootstrap-server", cluster.bootstrapServers(),
+                "--alter", "--entity-type", "client-metrics", "--entity-name", "test");
+        try (Admin client = cluster.createAdminClient()) {
+            Throwable e = assertThrows(ExecutionException.class, () ->
+                    alterConfigWithKraft(client, Optional.of(defaultBrokerId), singletonMap("interval.ms", "aaa")));
+            Assertions.assertTrue(e.getMessage().contains("InvalidConfigurationException"));
```

Comment:
There is a static import `import static org.junit.jupiter.api.Assertions.assertTrue`, so `Assertions` could be omitted.

Reply:
`InvalidConfigurationException.class.getSimpleName()` might be better instead of string literal.

Reply:
Thanks for review, updated!